templates/search_results.hbs

```diff
@@ -39,12 +40,13 @@
           {{#is current_filter.identifier 'community'}}
             <h3 class="collapsible-sidebar-title sidenav-title">{{t 'filter_by_topic'}}</h3>
           {{/is}}
-          <ul>
+          <ul class="multibrand-filter-list multibrand-filter-list--collapsed">
             {{#each subfilters}}
               <li>
                 <a href="{{url}}" class="sidenav-item" aria-selected="{{selected}}">{{name}} ({{count}})</a>
               </li>
             {{/each}}
+            <a tabindex="0" class="see-all-filters-trigger" aria-hidden="true" title="{{t 'see_more'}}">{{t 'see_more'}}</a>
```

Comment:
Please use `button` instead for `a`.

And `tabindex="0"` shouldn't be necessary.

Reply:
same here